petabvis/options_window.py

```diff
@@ -0,0 +1,337 @@
+            # bar plots can also be in the list
+            if isinstance(plot, VisSpecPlot):
+                for line in plot.dotted_lines + plot.dotted_simulation_lines:
+                    if state == Qt.Checked:
+                        line.show_points()
+                    else:
+                        line.hide_points()
+
+    def error_box_changed(self, state):
+        """
+        Add error bars when ticked, otherwise
+        remove them.
+        """
+        for plot in self.plots:
+            # bar plots can also be in the list
+            if isinstance(plot, VisSpecPlot):
+                for line in plot.dotted_lines + plot.dotted_simulation_lines:
+                    if state == Qt.Checked:
+                        line.show_errors()
+                    else:
+                        line.hide_errors()
+
+
+class CorrelationOptionMenu(QtGui.QMainWindow):
+    """
+    Option menu for the correlation plot.
+    """
+    def __init__(self, vis_spec_plots):
+        super(CorrelationOptionMenu, self).__init__()
+        self.resize(150, 80)
+        self.setWindowTitle("Correlation Options")
+        self.plots = vis_spec_plots
+        layout = QVBoxLayout()
+
+        self.cbox = QComboBox()  # dropdown menu to select plots
+        self.cbox.currentIndexChanged.connect(lambda x: self.index_changed(x))
+        self.cbox.addItems(["DatasetId", "ObservableId",
+                            "SimulationConditionId"])
+        self.names_lookup = {"DatasetId": "dataset_id",
+                             "ObservableId": "observable_id",
+                             "SimulationConditionId":
+                                 "simulation_condition_id"}
+        self.description = QLabel("Color points by:")
+
+        layout.addWidget(self.description)
+        layout.addWidget(self.cbox)
+        widget = QWidget()
+        widget.setLayout(layout)
+        self.setCentralWidget(widget)
+
+    def index_changed(self, i: int):
+        """
+        Color the correlation points based on the
+        selected id.
+        """
+        for plot in self.plots:
+            color_by = self.names_lookup[self.cbox.itemText(i)]
+            plot.add_points(plot.overview_df, color_by)
+
+
+class OverviewPlotWindow(QtGui.QMainWindow):
+    """
+    Window for plotting and displaying an overview plot.
+    """
+    def __init__(self, measurement_df, simulation_df):
+        super(OverviewPlotWindow, self).__init__()
+        self.measurement_df = measurement_df
+        self.simulation_df = simulation_df
+        self.resize(1000, 500)
+        self.setWindowTitle("Overview plot")
+        self.plot_by_label = QLabel("Plot overview by:")
+        self.plot_by = "ObservableId"
+        self.ids_label = QLabel("ObservableId:")
+
+        # plot
+        self.overview_plot = pg.PlotItem(title="Overview")
+        self.plot_widget = pg.GraphicsLayoutWidget(show=True)
+        self.overview_plot.setLabel("left", "r-squared")
```

Comment:
Unfortunately, $R^2$ does not work.

Reply:
Thanks for checking, looks like the "way" to do it is with HTML, so maybe the `<sup>` tag could be used. Anyway, not so important so fine to leave as is (change to "R squared" -- or a constant which is equal to this)

https://github.com/pyqtgraph/pyqtgraph/issues/1425

Reply:
Simply `²` should work - not?

Reply:
> Simply `²` should work - not?

Agreed, would go with this :+1:

Reply:
> Simply `²` should work - not?

Did not work for me either. I tried it by just putting `²` in quotes. Do I have to escape any character?

Reply:
> > Simply `²` should work - not?
> 
> Did not work for me either. I tried it by just putting `²` in quotes. Do I have to escape any character?

What happens? Does it appear as `R2` instead of `R²`? Could try e.g. `R_squared_readable = "R"+u"\u00B2"`, which is the Python Unicode encoding.

Reply:
Well, it just appears as R`²` (not as displayed here but as written in the string).
I think I will just leave it as "R squared" for this PR.